core/src/core-api/featuretoggles.js

```diff
@@ -54,7 +46,37 @@ class LuigiFeatureToggles {
    * @example Luigi.featureToggles().getActiveFeatureToggleList();
    */
   getActiveFeatureToggleList() {
+    return [...get(this.featureToggleList)].filter(ft => !ft.startsWith('!'));
+  }
+
+  /**
+   * Get a list of active feature toggles
+   * @memberof FeatureToggles
+   * @return {Array} of feature toggles
+   * @example Luigi.featureToggles().getFeatureToggleList();
+   */
+  getFeatureToggleList() {
     return [...get(this.featureToggleList)];
   }
+
+  isValid(featureToggleName) {
+    if (isString(featureToggleName)) return true;
+
+    console.warn('Feature toggle name is not valid or not a type of string');
+    return false;
+  }
+
+  isDuplicateOrDisabled(featureToggleName) {
+    if (get(this.featureToggleList).includes(`${featureToggleName}`)) {
```

Comment:
```suggestion
    if (get(this.featureToggleList).includes(featureToggleName)) {
```
We could remove the back-ticks here I suppose, in case it is not necessary.